packages/formdata-event/ts_src/environment/element.ts

```diff
@@ -18,4 +18,5 @@ export const methods = {
   hasAttribute: prototype.hasAttribute,
   removeAttribute: prototype.removeAttribute,
   setAttribute: prototype.setAttribute,
+  toggleAttribute: prototype.toggleAttribute,
```

Comment:
I don't think this is used anywhere but also, meh, I'd not worry about adding `toggleAttribute` support to the the formdata event polyfill: it was never released and might already be obsolete, IIRC.

Reply:
I was just adding it everywhere there were attribute methods... seem ok just in case this did get used in the future and toggleAttribute didn't work. If we remove the whole package, that's fine too.

Reply:
Adding it here doesn't do anything on its own though. This file is kind of like the `Native.ts` in the custom elements polyfill: it just captures the native stuff at the time the polyfill is installed so that it can be used by something else later.

https://github.com/webcomponents/polyfills/blob/4dc188e9afd53a3465cedcaefe0aea7da80b076f/packages/formdata-event/ts_src/environment/element.ts#L18

https://github.com/webcomponents/polyfills/blob/4dc188e9afd53a3465cedcaefe0aea7da80b076f/packages/formdata-event/ts_src/environment_api/element.ts#L12-L16

https://github.com/webcomponents/polyfills/blob/4dc188e9afd53a3465cedcaefe0aea7da80b076f/packages/formdata-event/ts_src/dispatch_formdata_for_submission.ts#L25

https://github.com/webcomponents/polyfills/blob/4dc188e9afd53a3465cedcaefe0aea7da80b076f/packages/formdata-event/ts_src/dispatch_formdata_for_submission.ts#L106

Reply:
removed